code/datums/keybinding/xeno.dm

```diff
@@ -737,3 +737,21 @@
 		return
 	var/mob/living/carbon/xenomorph/xeno = user.mob
 	xeno.vent_crawl()
+
+/datum/keybinding/xeno/psychic_shield
+	name = "Psychic shield"
+	full_name = "Warlock: Psychic shield"
+	description = "Channel our psychic power into a shield to reflect projectiles."
+	keybind_signal = COMSIG_XENOABILITY_PSYCHIC_SHIELD
+
+/datum/keybinding/xeno/psychic_blast
+	name = "Psychic blast"
+	full_name = "Warlock: Psychic blast"
+	description = "Unleash a devestating blast of psychic energy after a short charge-up."
+	keybind_signal = COMSIG_XENOABILITY_PSYCHIC_BLAST
+
+/datum/keybinding/xeno/psychic_crush
+	name = "Psychic crush"
+	full_name = "Warlock: Psychic crush"
+	description = "Channel our psychic power before unleashing it to crush enemies over an area."
```

Comment:
```suggestion
	description = "Channel an expanding AOE crush effect, activating it again pre-maturely crushes enemies over an area."
```

Reply:
Not to mention you do not have a keybind for psy lance.

Reply:
psylance is the same ability as psyblast, you toggle between them.